src/components/plots/DotPlot.jsx

```diff
@@ -28,7 +28,7 @@ const DotPlot = (props) => {
       return (
         <PlatformError
           error={cellSetsError}
-          reason={cellSetsError.message}
+          reason={cellSetsError}
```

Comment:
Error and reason are the same?

Reply:
original cellSetsError was an object. However, everywhere else in the code the errors stored in redux are the message (strings) so I unified them to all be string.